### PR TITLE
Report Generation

### PR DESCRIPTION
**Pull request by aibobrov**

### CODE REVIEW

File: src/main/kotlin/org/jetbrains/research/ml/coding/assistant/report/HintReportGenerator.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+""".trimIndent()
+        outputStream.write(text.toByteArray())
+    }
+
+    private fun fetchCode(vertex: SolutionSpaceVertex): String {
+        return codeRepository.getCode(vertex.info.first().id)
+    }
+}
+
+class CompositeMarkdownHintReportGenerator(private val generator: HintReportGenerator) {
```
Comment (nbirillo) [inline src/main/kotlin/org/jetbrains/research/ml/coding/assistant/report/HintReportGenerator.kt:57]: Please, add some description

**Review by nbirillo — CHANGES_REQUESTED** [review]
Great!

**Review by elena-lyulina — APPROVED** [review]
Cool!

**Review by elena-lyulina — APPROVED** [review]
Approve, thanks

**Pull request merged**